Makefile

```
@@ -1,4 +1,4 @@
-GO_VERSION = 1.18.9
+GO_VERSION = 1.20.5
```

Locally built client from .debs created direct goreleaser briefly smoke tested on Linux x86_64, no problems noticed.

However, the build using `make dist` results in dependencies to a recent glibc, 1.20.5 is an alias for 1.20.5-bookworm. These builds don't work e.g. on Ubuntu 20.04:
```shellsession
$ sshi req
sshi: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by sshi)
sshi: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by sshi)
```

For example this alleviates some, and makes the binary work on Ubuntu 20.04, but it's not a complete fix.
```suggestion
GO_VERSION = 1.20.5-bullseye
```

Instead of opening that can of worms, building without cgo could be a better approach, i.e. `--env CGO_ENABLED=0 \` for the docker command.

---

Please test again. Disabling CGO is probably the way to go as that was the case with previous linux builds.

---

LGTM :+1: